tests/README.md

```diff
@@ -0,0 +1,12 @@
+# Test script for key servers
+
+```
+# if necessary
+nvm install 20
+nvm use 20
+
+pnpm i
+pnpm test
```

Comment:
let's add cmd params so it would be easier to use it (including in ci):
- network=mainnet|testnet
- object_ids=X,Y,Z